nautobot/dcim/models/device_components.py

```diff
@@ -397,6 +398,14 @@ class PowerPort(ModularComponentModel, CableTermination, PathEndpoint):
         validators=[MinValueValidator(1)],
         help_text="Allocated power draw (watts)",
     )
+    power_factor = models.DecimalField(
```

Comment:
Ignorant question - are there use cases for setting this differently per-power-port? Or should it be set at a higher level, e.g. per Device or even globally?

Reply:
There *may* be use cases for setting it individually - okay to leave for now.